beacon-chain/db/iface/interface.go

```diff
@@ -58,15 +58,15 @@ type ReadOnlyDatabase interface {
 	// Powchain operations.
```

Comment:
Indeed, batched reads and writes are the way to go here, so the interface methods are adapted to reflect that improvement